docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fast Fourier Transform (FFT)\n",
+    "The following gives background information that is helpful to understand how the Fast Fourier Transform (FFT) and the corresponding normalizations are defined in pyfar and how these are related to the concepts of energy and power signals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## FFT definition\n",
+    "The discrete Fourier spectrum of an arbitrary, but band-limited signal $x(n)$ is defined as $$ X(\\mu) = \\sum_{n=0}^{N-1} x(n) e^{-i 2 \\pi \\frac{\\mu n}{N}}$$ using a negative sign convention in the transform kernel $e^{-i 2 \\pi \\mu \\frac{n}{N}}$. Analogously, the discrete inverse Fourier transform is implemented as\n",
+    "\n",
+    "$$ x(n) = \\frac{1}{N} \\sum_{\\mu=0}^{N-1} X(\\mu) e^{i2\\pi\\frac{\\mu n}{N}}$$\n",
+    "\n",
+    "Pyfar uses a DFT implementation for purely real-valued time signals resulting in Fourier spectra with complex conjugate symmetry for negative and positive frequencies $X(\\mu) = X(-\\mu)^*$. As a result, the left-hand side of the spectrum is discarded, yielding $X_R(\\mu) = X(\\mu)~\\forall 0 \\le \\mu \\le N/2$. Complex valued time signals can be implemented, if required."
```

Comment:
Maybe we should formulate this in a way that already fits the soon to be introduced complex Signals, e.g., 
```suggestion
    "Real-valued time signals resulting in Fourier spectra with complex conjugate symmetry for negative and positive frequencies $X(\\mu) = X(-\\mu)^*$. In this case, the left-hand side of the spectrum is discarded, and only the right-hand side is saved $X_R(\\mu) = X(\\mu)~\\forall 0 \\le \\mu \\le N/2$. "

    Additionally the term `N/2` shoud be surrounded by floor operators to be corect for signals with an uneven number of samples.
```

Reply:
Here, we are using mu for frequency, but k in the table below. I would be in favor of using k everywhere and mentionin that it is the normalized frequency, \omega_k = 2 \pi k / N

Reply:
The 'right sided' spectrum X_r is termed 'single-sided' in the table below (X_SS). I would be in favor of following the 'single-sided' convention everywhere. This would be consistent with Jens' tutorial that we are referring to.

Reply:
> The 'right sided' spectrum X_r is termed 'single-sided' in the table below (X_SS). I would be in favor of following the 'single-sided' convention everywhere. This would be consistent with Jens' tutorial that we are referring to.

Are you sure? Isn't the difference between right-sided and single-sided actually the difference between `'none'` and `'unitary'` normalization (factor of 2)?

Reply:
Hm, yes that would be one way of looking at it. Should we add X and/or X_R to the column 'none' in the table that introduces the normalizations to make this more clear?

Reply:
From my perspective, it is more a matter of how the data is saved (right sided only vs. full spectrum). Therefore, I removed the X_R, giving only the explanation.